pom.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Typical project structure is that we put all object model classes in one dir, services in another dir, repositories in another itp. Lets also do some refactor and create base entity class, every object model can extend it, see https://github.com/spring-projects/spring-petclinic/tree/main/src/main/java/org/springframework/samples/petclinic/model

Reply:
done